__tests__/endpoints.test.js

```diff
@@ -133,3 +130,75 @@ describe("GET /api", () => {
 			});
 	});
 });
+
+describe("PATCH: /api/articles/:article_id ", () => {
+	test("PATCH: 200 increase votes ", () => {
+		return request(app)
+			.patch("/api/articles/1")
+			.send({ inc_votes: 50 })
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toMatchObject({
+					article: {
+						article_id: 1,
+						title: "Living in the shadow of a great man",
+						topic: "mitch",
+						author: "butter_bridge",
+						body: "I find this existence challenging",
+						created_at: "2020-07-09T20:11:00.000Z",
+						votes: 150,
+						article_img_url:
+							"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+					},
+				});
+			});
+	});
+	test("PATCH: 200 decrease votes to a negative number", () => {
+		return request(app)
+			.patch("/api/articles/1")
+			.send({ inc_votes: -500 })
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toMatchObject({
+					article: {
+						article_id: 1,
+						title: "Living in the shadow of a great man",
+						topic: "mitch",
+						author: "butter_bridge",
+						body: "I find this existence challenging",
+						created_at: "2020-07-09T20:11:00.000Z",
+						votes: -400,
+						article_img_url:
+							"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+					},
+				});
+			});
+	});
+	test("PATCH: 404 if the client sends a non-number as the value", () => {
+		return request(app)
+			.patch("/api/articles/1")
+			.send({ inc_votes: "tony" })
+			.expect(404)
+			.then(({ body }) => {
+				expect(body).toMatchObject({ msg: "Bad request" });
+			});
+	});
+	test("PATCH: 404 if the client send an incorrect article_id", () => {
+		return request(app)
+			.patch("/api/articles/404")
+			.send({ inc_votes: 10 })
+			.expect(404)
+			.then(({ body }) => {
+				expect(body).toMatchObject({ msg: "resource not found" });
+			});
+	});
+	test("PATCH: 404 if the client send an SQL injection", () => {
+		return request(app)
+			.patch("/api/articles/1")
+			.send({ inc_votes: ";DROP Table comments;" })
+			.expect(404)
+			.then(({ body }) => {
+				expect(body).toMatchObject({ msg: "Bad request" });
+			});
+	});
```

Comment:
Nice tests 👍 
You'll need to add one more. A 400 - /api/articles/notanumber test :)